### ISSUE TITLE
Sidebar scrolling

### ISSUE THREAD

**Issue by ghost** [Snoobyx/protonmail-material-themes#2]
The left hand side bar has scrolling bars horizontally.

**Comment by delacrixmorgan** [comment]
Yeah, I am having the same issue. It's the sizing issue that made the unnecessary sidebar horizontally scrolling issue. 

Tried it on both the latest Firefox and Chrome with same results.

Firefox - 58.0
Chrome - 64.0.3282.119
